jbpm-wb-integration/jbpm-wb-integration-client/src/main/resources/org/jbpm/workbench/wi/client/i18n/Constants.properties

```diff
@@ -94,5 +94,13 @@ InvalidPom=Corrupted file content
 UploadFailed=Upload of service task failed
 InternalUploadError=Unable to add service task due to internal error
 AddTaskSuccess=Service tasks successfully added - 
+SkippedMessage=Service tasks successfully skipped - 
```

Comment:
What does it mean successfully skip? I'd say "Service tasks skipped - "

Reply:
it means "2 when user uploads a new jar, and the jar have duplicates name ST, we skip all new duplicates name ST of the new jar."

Reply:
ok